### DISCUSSION TITLE
Add mechanic to capture a monster instead of killing it, with items, objectives and maybe a new class around it.

### DISCUSSION THREAD

**Discussion by terrykwan1** [FakeFishGames/Barotrauma#6358] [Ideas and suggestions]
### Basic Pitch
Similar to beacons there can be an optional objective or valuation around captured creatures.

That would be to capture a crawler/thresher/mudraptor, or any other small creature.

### Mechanics and Implications
To do this players would need to use **chemicals or non-lethal weapons to incapacitate the creature**, then restrain it with some kind of **net/cage** that can be dragged back to the ship.

I think this would increase the amount of possible interactions on the ship by a good amount.

- There is a possibility that if the creature recovers too much it will struggle and possibly break its containment in the ship or maybe a traitor lets it loose. Cages could be fixed with mechanical engineering, a terrible fix could release the creature.
- If drugs can have separate effects on creatures that would be cool or just add more drugs/poisons that work better on creatures. Gives more reasons to use other poisons besides being a traitor or trolling other players.
- Adds extra method of generating money or rep, research stations will value captured creatures more.

Could lead to more involved mechanics like taming creatures, processing creatures for food/organs/leather, trading tamed creatures, and even a new class.

### Xenobiologist

Who specializes in harvesting alien bodies for materials and specializes in healing/operations involving aliens instead of humans so that it is distinct from medics. Gains skills for butchering/dissecting creatures or crafting certain Xeno toxins.

Capturable creatures can have their own health screen that is roughly a head, torso and tail. If greater efforts are made they can even have extra sections that are drawn if you have more levels of Xenobiologist.

If the organ thing is done maybe we could even let medics harvest organs from people and sell them or use them in other people?

### Submarine parts

Logically this can also lead to big and small built-in containment cells that may or not be filled with water that can be added to subs and subs design around capturing and trafficking creatures.

Breaching such containment could cause floods as the aquarium drains into the sub and cause creatures to slip and slide around your deck biting people.

**Comment by ThatChaFella** [comment]
That sounds like itd be pretty fun and Id love to see it either in base game or as a mod.Great idea

**Comment by ViktorShield** [comment]
Finally some use to submarine prisons...
